diagnostic.md

```diff
@@ -7,40 +7,44 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git clone git@github.com:aeabbott/git-diagnostic.git
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+git status
```

Comment:
`git status` is good to run but here we were looking for creating a new branch to do your local work on.